src/metabase/channel/template/handlebars.clj

```diff
@@ -0,0 +1,64 @@
+(ns metabase.channel.template.handlebars
+  (:require
+   [clojure.walk :as walk]
+   [metabase.util :as u])
+  (:import
+   (java.io IOException)
+   (com.github.jknack.handlebars
+    Handlebars Helper Template)
+   (com.github.jknack.handlebars.io
+    TemplateLoader ClassPathTemplateLoader CompositeTemplateLoader TemplateSource)
+   (com.github.jknack.handlebars.cache
+    ConcurrentMapTemplateCache)))
+
+(deftype AppDBTemplateSource [^String name ^String the-content last-modified]
+  TemplateSource
+  (content [_ _] the-content)
+  (filename [_] name)
+  (lastModified [_] last-modified))
+
+(deftype AppDBLoader []
+  TemplateLoader
+  (sourceAt [_this _template-name]
+    (throw (IOException. "Not implemented"))
+    #_(AppDBTemplateSource. template-name "hello {{name}}" (System/currentTimeMillis))
```

Comment:
I know this is still stubbed, but not sure about current time for age. Perhaps it makes more sense to use null? Ideally it would be when the user last upgraded to a version with template, talk about overly complicated though 😅

Reply:
yea, I'll need to play with this to make sure it works correctly with caching. this will not be included in this PR tho, was just playing with a custom loader.